Blueprints/BlueprintDefinitions/Msbuild-NETCore_2_1/CustomRuntimeFunction-FSharp/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -0,0 +1,43 @@
+# AWS Lambda Custom Runtime Function Project
+
+This starter project consists of:
+* Function.fs - contains a Main method function that starts the bootstrap, and a single function handler
```

Comment:
"method function" sounds weird

Reply:
looks like main should be lower case in F#

Reply:
Fixed